src/main/java/org/broadinstitute/hellbender/tools/spark/sv/ContigsCollection.java

```diff
@@ -85,50 +49,52 @@ public String toPackedFasta(){
      * @param packedFastaLine
      */
     @VisibleForTesting
-    protected static ContigsCollection fromPackedFasta(final String packedFastaLine) {
+    protected static ContigsCollection fromPackedFasta(final long assemblyID, final String packedFastaLine) {
```

Comment:
I guess my concern about not adding things until we need them also applies here, then. In this case I guess it might be alright -- at least we are setting the value, rather than leaving it untouched.